### DISCUSSION TITLE
Can not connect with my GitHub

### DISCUSSION THREAD

**Discussion by Hinrick** [coder/code-server#2497] [Q&A]
Not sure if this is the right place to ask? If not, please let me know, I will remove this discussion.

I tried to do `code-server --link` my very first time and get the error using one of my Google accounts.
![螢幕截圖 2020-12-21 下午2 28 11](https://user-images.githubusercontent.com/28645974/102746218-c2f72980-4398-11eb-8341-63c28608ae74.png)
After, I tried to delete the VM instance on GCP and remake one using A account but had faced the same issue.
After, I use my other google account and had faced the same account.

Eventually, I asked one of my friend to use his Github and Google account and it works?

I'm thinking this is some cache issue of my Github account.

Can anyone helps?

**Comment by code-asher** [comment]
If you run code-server with `--log debug` are there are errors in the
output when you try to log in?
